frappe/core/doctype/version/version.js

```diff
@@ -9,4 +9,22 @@ frappe.ui.form.on("Version", "refresh", function (frm) {
 			docname: frm.doc.docname,
 		});
 	});
+	if (frm.doc.complete_snapshot) {
+		frm.add_custom_button(__("Restore"), function () {
+			frappe.call({
+				method: "frappe.core.doctype.version.version.restore",
+				args: {
+					version: frm.doc.name,
+				},
+				callback: function (r) {
+					if (!r.exc) {
+						frappe.show_alert({
+							message: __("Restored"),
```

Comment:
We should show a link to document at least. `Version` doctype doesn't have dynamic link so it's not possible to go to doc from version.

Reply:
This will be removed, based on cherry-picks from @safwansamsudeen 's  PR